README.md

```diff
@@ -6,14 +6,13 @@ acbuild is a command line utility to build and modify App Container Images
 
 ## Rationale
 
-Dockerfiles are powerful and feature useful concepts such as build layers, 
-controlled build environment. At the same time, they lack flexibility 
+Dockerfiles are powerful, and feature useful concepts such as build layers and
```

Comment:
I'd be in favor of not mentioning them at all. That text is leftover from the original README.

Perhaps:
We wanted a powerful tool for manipulating ACIs, that made it easy to iteratively build images both from scratch and on top of existing images, and would play nicely with popular Linux tools like the shell and `Makefile`s.

Might need rephrasing and/or more points tossed in there though.